src/coreclr/inc/corcompile.h

```diff
@@ -161,10 +161,10 @@ enum CORCOMPILE_FIXUP_BLOB_KIND
 
     ENCODE_CHECK_IL_BODY,                           /* Check to see if an IL method is defined the same at runtime as at compile time. A failed match will cause code not to be used. */
     ENCODE_VERIFY_IL_BODY,                          /* Verify an IL body is defined the same at compile time and runtime. A failed match will cause a hard runtime failure. */
-
+    
     ENCODE_MODULE_HANDLE                = 0x50,     /* Module token */
-    ENCODE_STATIC_FIELD_ADDRESS,                    /* For accessing a static field */
-    ENCODE_MODULE_ID_FOR_GENERIC_STATICS,           /* For accessing static fields */
+    // obsolete: ENCODE_STATIC_FIELD_ADDRESS = 0x51, /* For accessing a static field */
+    ENCODE_MODULE_ID_FOR_GENERIC_STATICS = 0x52,    /* For accessing static fields */
```

Comment:
```suggestion
    ENCODE_MODULE_ID_FOR_GENERIC_STATICS, /* For accessing static fields */
```
These are not part of the stable R2R format. They should be all deleted (it can be separate change).